daffodil-test/src/test/resources/org/apache/daffodil/usertests/nameDOB_test.tdml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Missing apache header, causing the rat check to fail.

Reply:
Done